tests/CarbonSleeper/SleepTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Tests\Carbon;
```

Comment:
This should be put in the `tests/CarbonSleeper` folder and the filename should be `Tests\CarbonSleeper\SleepTest`